packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -541,15 +500,14 @@ export class SDKClient {
           `${requestIdPrefix} Invalid response format, expected record availability: ${JSON.stringify(resp)}`,
         );
       }
-      const shouldLimit = this.hbarLimiter.shouldLimit(currentDateNow, SDKClient.recordMode, transactionName);
-      if (shouldLimit) {
-        throw predefined.HBAR_RATE_LIMIT_EXCEEDED;
-      }
+      const transactionRecord = await new TransactionRecordQuery()
+        .setTransactionId(resp.transactionId!)
+        .setValidateReceiptStatus(false)
```

Comment:
Missing the nodeIds selection.
Was this intentionally left out?
I believe the optimization allows for you to get it from the node you submitted to for shorter latency.